api.swagger.yaml

```diff
@@ -6116,6 +6133,18 @@ components:
         default: 25
       description: The numbers of items to return.
   schemas:
+    pdf-export-completion-schema:
+      type: object
+      properties:
+        token:
```

Comment:
This may be fine but if we decide to move the token to a header for the other endpoint we should do it here too for consistency

Reply:
Change applied.